chain-impl-mockchain/src/testing/address.rs

```diff
@@ -108,7 +135,7 @@ impl AddressData {
     }
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, PartialEq, PartialOrd)]
```

Comment:
Good for `PartialEq`, dodgy for `PartialOrd`.

Reply:
it's some leftovers from refactoring. removed